mailpoet/assets/js/src/automation/editor/api-error-handler.tsx

```diff
@@ -16,6 +17,12 @@ export const registerApiErrorHandler = (): void =>
       } catch (error) {
         const errorObject = error as ApiError;
         const status = errorObject.data?.status;
+        const code = errorObject.code;
+
+        if (code === 'mailpoet_automation_workflow_not_valid') {
+          dispatch(storeName).setErrors({ steps: errorObject.data.errors });
+          return undefined;
```

Comment:
So `<Error />` pops up, but because of the early return no `Error message` which showed up will be shown. I guess it's a good thing and intended, but wanted to point that out.

Reply:
Yes, in this case, I think that's correct because, in the designs, there is no other "main" error message apart from the `X issues` chip. So one more mention of @NeosinneR whether we should consider also showing an in-editor notice saying sth like `Could not activate workflow because some steps are not fully set up...`?

Reply:
Without the in-editor notice, would the user just see the `2 issues` chip in the header with a popup of not set steps?

<img width="1440" alt="Editor_Default" src="https://user-images.githubusercontent.com/470616/195553961-94ae7047-4287-493b-80f1-6074fa0f0bbd.png">

Reply:
@NeosinneR Yes, that's the current state. The popup auto-opens.

Reply:
Let's keep it like that, without global notice. I'd guess we'll receive feedback if that's not adequate.